src/Uno.Toolkit.UI/Controls/NavigationBar/AppBarButtonRenderer.iOS.cs

```diff
@@ -93,18 +93,18 @@ protected override IEnumerable<IDisposable> Initialize()
 				new[] { AppBarButton.IsInOverflowProperty }
 			);
 
-			Native.Clicked += OnNativeClicked;
+			Native!.Clicked += OnNativeClicked;
 			yield return Disposable.Create(() => { Native.Clicked -= OnNativeClicked; });
 		}
 
-		private void OnAppBarButtonWrapperParentChanged(object sender, EventArgs e)
+		private void OnAppBarButtonWrapperParentChanged(object? sender, EventArgs e)
 		{
 			// Even though we set the NavigationBar as the parent of the _appBarButtonWrapper,
 			// it will change to the native control when the view is added.
 			// This control is the visual parent but is not a DependencyObject and will not propagate the DataContext.
 			// In order to ensure the DataContext is propagated properly, we restore the NavigationBar
 			// parent that can propagate the DataContext.
-			if (!ReferenceEquals(_appBarButtonWrapper?.Parent, Element.Parent))
+			if (!ReferenceEquals(_appBarButtonWrapper?.Parent, Element!.Parent))
```

Comment:
@Xiaoy312 I'm not sure if throwing on `null` Element is the appropriate action. The `Renderer` constructor takes a non-null element, but we create a `WeakReference` for it.

The `WeakReference` seems to indicate that it's acceptable for the element to be GC'ed, which in turn means that we should handle null gracefully without exceptions.
Otherwise, if null is not allowed, we shouldn't use a WeakReference?

Could you please clarify? Thanks!

Reply:
The line in this file has since shifted since I last reviewed.
This was originally written for the `Render` method regarding these two:
```
// Icon & Content
var native = Native;
var element = Element;
```
There is no null-handling here. Since we don't know how to gracefully handling that, null-coalesce-throw is at worse retaining original behavior.

Reply:
It's somewhat unclear to me why we use WeakReference in this case. But I'm going to retain the original behavior by explicitly throwing InvalidOperationException for null Element.